osu.Server.DifficultyCalculator/osu.Server.DifficultyCalculator.csproj

```diff
@@ -12,16 +12,16 @@
     <ItemGroup>
         <PackageReference Include="Dapper" Version="2.0.123" />
         <PackageReference Include="Dapper.Contrib" Version="2.0.78" />
-        <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="3.1.0" />
-        <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.0" />
-        <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="6.0.0" />
+        <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="4.0.1" />
+        <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1" />
+        <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="6.0.1" />
         <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
-        <PackageReference Include="MySqlConnector" Version="2.0.0" />
-        <PackageReference Include="ppy.osu.Game" Version="2021.1113.0" />
-        <PackageReference Include="ppy.osu.Game.Rulesets.Osu" Version="2021.1113.0" />
-        <PackageReference Include="ppy.osu.Game.Rulesets.Taiko" Version="2021.1113.0" />
-        <PackageReference Include="ppy.osu.Game.Rulesets.Catch" Version="2021.1113.0" />
-        <PackageReference Include="ppy.osu.Game.Rulesets.Mania" Version="2021.1113.0" />
+        <PackageReference Include="MySqlConnector" Version="2.1.10" />
+        <PackageReference Include="ppy.osu.Game" Version="2022.605.1" />
+        <PackageReference Include="ppy.osu.Game.Rulesets.Osu" Version="2022.605.1" />
+        <PackageReference Include="ppy.osu.Game.Rulesets.Taiko" Version="2022.605.1" />
+        <PackageReference Include="ppy.osu.Game.Rulesets.Catch" Version="2022.605.1" />
+        <PackageReference Include="ppy.osu.Game.Rulesets.Mania" Version="2022.605.1" />
```

Comment:
You can't use these packages.

I would rather not update packages in a backport because I don't know what/if anything else required fixes.

Reply:
What's the reasoning? Are the versions supposed to be pinned here to the current live or something?

Reply:
Correct, but also see my comment below.

Reply:
Oh, this isn't even on master......................................

Reply:
I can't force push back so i'll just revert my change.